packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -1139,6 +1139,17 @@ new ecs.Ec2Service(this, 'EC2Service', {
 });
 ```
 
+### Cluster Default Provider Strategy
+When the service does not have a capacity provider strategy, cluster's default capacity provider strategy will be used.
```

Comment:
Are there any caveats to this? Would you ever be able to use a Fargate service and fall back to the default capacity provider strategy? What happens when someone creates a FargateService on a cluster with a default ec2 capacity provider strategy?